File: Cappacita/ex_12.js
// Altere o conteúdo chiclete do array doces para doce de coco

doces = ["bala", "pirulito", "chiclete", "jujuba", "chocolate"]

console.log("Antes: " + doces)
for (doce in doces){
    if(doces[doce] == "pirulito"){
        doces[doce] = "doce de leite"
    }else if(doces[doce] == "chiclete"){
        doces[doce] = "doce de coco"
    } 
}
console.log("Depois: " + doces)

File: Cappacita/ex_12.py
doces = ["bala", "pirulito", "chiclete", "jujuba", "chocolate"]

print(f'Antes: {doces}')
for num, doce in enumerate(doces):
    if doces[num] == "pirulito":
        doces[num] = "doce de leite"
    elif doces[num] == "chiclete":
        doces[num] = "doce de coco"    
print(f'Depois: {doces}')